unitypackageのインポート時に警告がでる。

以下のコードを見直す。
Assets/VketBoothValidator/Editor/Rules/G_ComponentLimitation/G07_AnimatorRule.cs(55,37): warning CS0219: The variable `curve' is assigned but its value is never used

--- a/VketBoothValidator/Assets/VketBoothValidator/Editor/Rules/G_ComponentLimitation/G07_AnimatorRule.cs
+++ b/VketBoothValidator/Assets/VketBoothValidator/Editor/Rules/G_ComponentLimitation/G07_AnimatorRule.cs
@@ -52,7 +52,6 @@ namespace VketTools
                         {
                             foreach (var binding in AnimationUtility.GetCurveBindings(clip))
                             {
-                                var curve = AnimationUtility.GetEditorCurve(clip, binding);
                                 if (binding.path.StartsWith("../"))
                                 {
                                     dirtFlg = true;

